From 32154125ac74522126bc5eee67e99156c076280e Mon Sep 17 00:00:00 2001
From: Junwei Zhang <Jerry.Zhang@amd.com>
Date: Thu, 12 Jan 2017 14:11:26 +0800
Subject: [PATCH 07/39] amdgpu: add sparse flag for bo creatation v2

v2:
7987cf4262d2cf83f32eb5debef9aefa791c1505
update GEM flag bit for 64-bit extension

Change-Id: Ie291d391d0f68e7ff9028b3713808c38002d84c6
Signed-off-by: Junwei Zhang <Jerry.Zhang@amd.com>
Reviewed-by: Flora Cui <Flora.Cui@amd.com>
---
 include/drm/amdgpu_drm.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/drm/amdgpu_drm.h b/include/drm/amdgpu_drm.h
index b3ce535..afacd62 100644
--- a/include/drm/amdgpu_drm.h
+++ b/include/drm/amdgpu_drm.h
@@ -89,6 +89,8 @@ extern "C" {
 #define AMDGPU_GEM_CREATE_VRAM_CONTIGUOUS	(1 << 5)
 
 /* Hybrid specific */
+/* Flag that the memory should be in SPARSE resource */
+#define AMDGPU_GEM_CREATE_SPARSE		(1ULL << 29)
 /* Flag that the memory allocation should be pinned */
 #define AMDGPU_GEM_CREATE_NO_EVICT		(1ULL << 31)
 
-- 
2.7.4

